src/registrar/templates/application_withdraw_confirmation.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block title %}Withdraw request for {{ domainapplication.requested_domain.name }}{% endblock %}
+{% load static url_helpers %}
+
+{% block content %}
+<div class="grid-col desktop:grid-offset-2 desktop:grid-col-8">
```

Comment:
For alignment and spacing, this whole block should be wrapped in `<div class="grid-container">`

To clarify the container div would start right after `{% block content %}` and end right before `{% endblock %}`

This should resolve the padding issue.

Reply:
This one need to be tweaked just slightly — the news div does need to be inside the `{% block content %}`  (both the start and end of it) otherwise things getting a little off due to the wrapper.